src/app.ts

```diff
@@ -142,20 +150,20 @@ export class App {
       event.preventDefault();
     });
 
+    const enablePostEffects = this.store.state.preferences.enablePostEffects;
+
     const renderer = new WebGLRenderer({
-      // TODO we should not be using alpha: false https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/WebGL_best_practices#avoid_alphafalse_which_can_be_expensive
       alpha: true,
-      antialias: true,
-      depth: true,
-      stencil: true,
-      premultipliedAlpha: true,
-      preserveDrawingBuffer: false,
-      logarithmicDepthBuffer: false,
```

Comment:
Just to clarify, you have removed `premultipliedAlpha`, `preserveDrawingBuffer`, and `logarithmicDepthBuffer` because they use the default values?

https://threejs.org/docs/#api/en/renderers/WebGLRenderer.preserveDrawingBuffer

Reply:
Yep, exactly

Reply:
I hope we can have a separated clean up PR from a logic addition/change PR because review can be easier.